scripts/json-lint.py

```diff
@@ -33,7 +33,7 @@ def assert_list_image(img: dict):
     assert(isinstance(img['imageSrc'], str))
     image_exists(img['imageSrc'])
     assert(isinstance(img['imageAlt'], str))
-    assert(img['imageAlt'])
+    #assert(img['imageAlt'])
```

Comment:
This assertion is still important and should be modified to allow for empty strings:

```suggestion
    assert(isinstance(image['imageAlt'], str))
```

Also, if you are removing code in the future please delete it (don't just comment it out).